code/style.css

```diff
@@ -1,4 +1,191 @@
 html {
-  background: #0025ff;
+  background: #d9e5ec;
   color: #fff;
+  }
```

Comment:
you should reset default margins and paddings to avoid an empty frame around your page.